CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ Contributors:
 
 - Handle quoted values within test configs, such as `where` ([#3458](https://github.com/fishtown-analytics/dbt/issues/3458), [#3459](https://github.com/fishtown-analytics/dbt/pull/3459))
 
+### Under the hood
+
+- Improve default view and table materialization performance by checking relational cache before attempting to drop temp relations ([#3112](https://github.com/fishtown-analytics/dbt/issues/3112), [#3468](https://github.com/fishtown-analytics/dbt/pull/3468))
```

Comment:
Could you move this up to `v0.21`, and add yourself to the list of Contributors there?

This is a net-new change, so I'd prefer to have it in the next version, rather than trying to sneak it in between v0.20.0rc1 and final. We're going to be disciplined about keeping the v0.21 scope narrow and putting out prereleases early & often.

Reply:
totally. Didn't know where to put this, 0.21 makes more sense.